deploy/kubernetes/overlays/dev/kustomization.yaml

```diff
@@ -1,5 +1,22 @@
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
 bases:
 - ../../base
 patches:
-- controller_images.yaml
-- node_images.yaml
+- controller_add_snapshotter.yaml
+- controller_always_pull.yaml
+- node_always_pull.yaml
+images:
+- name: REPLACEME/gcp-compute-persistent-disk-csi-driver
+  # Replace this with your private image name
+  newName: gcr.io/gke-release/gcp-compute-persistent-disk-csi-driver
+  newTag: "v0.4.0-gke.0"
+- name: REPLACEME/csi-provisioner
+  newName: quay.io/k8scsi/csi-provisioner
```

Comment:
Why do we use quay here but GCR elsewhere?

Reply:
We have a bit of a chicken and egg problem.  Currently in order to test a new deployment, we need to modify the staging deployment here, check it in, and then look at the staging test results.  

Ideally, I want to be able to validate a new deployment in a pull job first before needing to check it in.  So I want to use the dev overlay for that, but the pull jobs cannot access gke-staging images, so we need to use public images.

Reply:
Maybe I'll rename this template to pull instead of dev to make it more obvious what it's purpose is for